### ISSUE TITLE
Chuẩn bị cho chương Multilayer Perceptrons

### ISSUE THREAD

**Issue by duythanhvn** [aivivn/d2l-vn#774]
- [x] 4. [Multilayer Perceptrons](chapter_multilayer-perceptrons/index_vn.md)
- [x] 4.1. [Multilayer Perceptrons](chapter_multilayer-perceptrons/mlp_vn.md)
- [x] 4.2. [Implementation of Multilayer Perceptron from Scratch](chapter_multilayer-perceptrons/mlp-scratch_vn.md)
- [x] 4.3. [Concise Implementation of Multilayer Perceptron](chapter_multilayer-perceptrons/mlp-gluon_vn.md)
- [x] 4.4. [Model Selection, Underfitting and Overfitting](chapter_multilayer-perceptrons/underfit-overfit_vn.md)
- [x] 4.5. [Weight Decay](chapter_multilayer-perceptrons/weight-decay_vn.md)
- [x] 4.6. [Dropout](chapter_multilayer-perceptrons/dropout_vn.md)
- [x] 4.7. [Forward Propagation, Backward Propagation, and Computational Graphs](chapter_multilayer-perceptrons/backprop_vn.md)
- [x] 4.8. [Numerical Stability and Initialization](chapter_multilayer-perceptrons/numerical-stability-and-init_vn.md)
- [x] 4.9. [Considering the Environment](chapter_multilayer-perceptrons/environment_vn.md)
- [x] 4.10. [Predicting House Prices on Kaggle](chapter_multilayer-perceptrons/kaggle-house-price_vn.md)

**Comment by tanduong** [comment]
@duythanhvn mình nhận tin nha. Thật ra thì mình khá bận nên hông dịch nhiều đâu, hôm nay cao hứng thôi. Thôi mình đi ngủ. G9!

**Comment by tiepvupsu** [comment]
@tanduong không nên ôm đồm quá bạn ạ. Việc này phụ thôi, khi nào có thời gian thì làm. Không nên để ảnh hưởng tới công việc chính.
